.travis.yml

```diff
@@ -0,0 +1,19 @@
+before_install:
```

Comment:
good work setting this up

Reply:
we probably will need to store the secure key in application.yml

Reply:
Nice! 
Doesn't Travis store the key for us and the key in the travis.yml file is a secure one?

Reply:
Yup, please ignore my previous comment, Stella is right, this is encrypted.